charts/linkerd2/templates/sp-validator-rbac.yaml

```
@@ -71,11 +71,15 @@ metadata:
 webhooks:
 - name: linkerd-sp-validator.linkerd.io
   namespaceSelector:
+    {{- if .Values.profileValidator.namespaceSelector }}
+    {{- toYaml .Values.profileValidator.namespaceSelector | trim | nindent 4 }}
+    {{- else }}
     matchExpressions:
     - key: config.linkerd.io/admission-webhooks
       operator: NotIn
       values:
       - disabled
+  {{- end }}
```

It would be safer to exclude your own namespace too. Once the kube-apiserver webhooks are modified, it will expect tap service to respond to pod creation events.

```suggestion
    - key: name
      operator: NotIn
       values:
       - {{ .Release.Namespace }}
     {{- end }}
```

---

I'd agree with this change, I in fact want something even stronger, but should we get that in as a separate PR?

---

That sounds good to me. We could use this PR as is without the namespace change